sql/sqlcheck/sqlcheck.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2021-present The Atlas Authors. All rights reserved.
+// This source code is licensed under the Apache 2.0 license found
+// in the LICENSE file in the root directory of this source tree.
+
+package sqlcheck
```

Comment:
This package resides in the top-level project because drivers will register their analyzers to it.